cmd/server/assets/realmkeys.html

```diff
@@ -0,0 +1,262 @@
+{{define "realmkeys"}}
+
+{{$realm := .realm}}
+{{$publicKeys := .publicKeys}}
+{{$csrfField := .csrfField}}
+
+<!doctype html>
+<html lang="en">
+<head>
+  {{template "head" .}}
+</head>
+
+<body>
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>Verification certificate key settings</h1>
+    <p>
+      View or edit the verification certificate signing keys for <strong>{{$realm.Name}}</strong> below.
+    </p>
+    
+    {{if not $realm.UseRealmCertificateKey}}
+      <!-- This is only shown if the realm is NOT set to use per-realm signing keys -->
+      <div class="card mb-3">
+        <div class="card-header">
+          System verification certificate information
+        </div>      
+        <div class="card-body">
+          <div class="alert alert-info" role="alert">
+                This information needs to be communicated to the operator
+                of you exposure notification key server in order to be able to utilize
+                verification certificates issued by this server.
+          </div>
+ 
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Issuer (iss):</label>
+            <div class="col-sm-9">
+              <input type="text" id="iss" class="form-control" value="{{.certIssuer}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Audience (aud):</label>
+            <div class="col-sm-9">
+              <input type="text" id="aud" class="form-control" value="{{.certAudience}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Certificate duration:</label>
+            <div class="col-sm-9">
+              <input type="text" id="certDuration" class="form-control" value="{{.certDuration}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Key ID (kid):</label>
+            <div class="col-sm-9">
+              <input type="text" id="certKeyID" class="form-control" value="{{.certKeyID}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Public key:</label>
+            <div class="col-sm-9">
+            {{if .certPublicKey}}
+            <div class="alert alert-secondary" role="alert">
+            <pre>{{.certPublicKey}}</pre>
+            </div>
+            {{else}}
+            <div class="alert alert-danger" role="alert">{{.certPublicKeyError}}</div>
+            {{end}}
+            </div>
+          </div>
+        </div>
+      </div>
+    {{end}}
+      
+    <div class="card mb-3">
+        <div class="card-header">
+          Realm signing key configuration
+        </div>
+        <div class="card-body">
+        {{if .supportsPerRealmSigning}}            
```

Comment:
This nested if is really hard to follow in Go templates. Can we either:

- Make use of `and`
- Break these out into smaller templates (using `define`) and include them so the nesting is more visual

?